src/main/java/com/datalogics/pdf/samples/forms/FillForm.java

```diff
@@ -171,28 +106,7 @@ public static void fillPdfForm(final URL inputFormUrl, final URL inputDataUrl, f
      */
     public static void fillAcroformFdf(final PDFDocument pdfDocument, final URL inputDataUrl, final URL outputUrl)
                     throws Exception {
-
-        // Open the input form data file.
-        final InputStream formStream = inputDataUrl.openStream();
-        final ByteReader formByteReader = new InputStreamByteReader(formStream);
-        final FDFDocument fdfDocument = FDFDocument.newInstance(formByteReader);
-
-        // Use the FDFService to get the form data into the PDF.
-        final FDFService fdfService = new FDFService(pdfDocument);
-        fdfService.importForm(fdfDocument);
-
-        // Run calculations on the AcroForm...only required before PDFJT 4.5.0
-        if (pdfjtIsBeforeVersion4()) {
-            FormFieldService.getAcroFormFieldManager(pdfDocument).runCalculateScripts();
-        }
-
-        // Run formatting...
-        FormFieldService.getAcroFormFieldManager(pdfDocument).runFormatScripts();
-        // And generate appearances.
-        AppearanceService.generateAppearances(pdfDocument, null, null);
-
-        // Save the file.
-        DocumentHelper.saveFullAndClose(pdfDocument, outputUrl.toURI().getPath());
+        FormImporter.fillAcroformFdf(pdfDocument, inputDataUrl, outputUrl);
```

Comment:
This has a similar issue to ``fillPdfForm()``: it's now a blind redirection to the ``FormImporter`` (this is also true of the following functions).

I'm not sure these methods are even necessary any more. Note that as of this comment, the only thing calling this method is the test code.